spec/system/task_spec.rb

```diff
@@ -1,25 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let!(:task) { create(:task, project: project) }
```

Comment:
associationを設定するとprojectの指定が不要になりますね。
```suggestion
  let!(:task) { create(:task) }
```

Reply:
taskのFactoryBotにassociationを記載することで、let内projectの指定を削除しました。

Reply:
50297bc